README.md

```diff
@@ -104,13 +104,13 @@ $ scp ubuntu@<INSTANCE>:images/livecd.ubuntu-cpc.squashfs .
 ## Customising the Built Images
 
 In order to customise the contents of the built images, you can provide
-a second argument to `generate_build_config.py`.  This should be the
-path to a shell script which will be run within the chroot after all
-other image building is complete.
+a parameter to `generate_build_config.py`, `--customisation-script`.
+The argument to this should be the path to a shell script which will be
+run within the chroot after all other image building is complete.
```

Comment:
I didn't pick this up before, but "after all the other image **building** is complete" seems confusing.  I would consider the binary hooks to produce disk1, vmdk, etc as image building as well.  Could this be "after all the other base image customization is complete"?